src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -124,6 +124,26 @@ public class RepositoryOptions extends OptionsBase {
       help = "Scale all timeouts related to http downloads by the given factor")
   public double httpTimeoutScaling;
 
+  @Option(
+      name = "http_connector_attempts",
+      defaultValue = "0",
```

Comment:
why not just make the default value 8? same for the other flag

Reply:
Done. Made the other one match the current behavior of no maximum by default.